src/test/java/games/strategy/engine/lobby/server/login/LobbyLoginValidatorTest.java

```diff
@@ -13,6 +13,8 @@
 
 import games.strategy.engine.lobby.server.LobbyServer;
 import games.strategy.engine.lobby.server.userDB.BadWordController;
+import games.strategy.engine.lobby.server.userDB.DbUser;
+import games.strategy.engine.lobby.server.userDB.HashedPassword;
 import games.strategy.engine.lobby.server.userDB.DbUserController;
```

Comment:
Checkstyle: "Wrong lexicographical order for 'games.strategy.engine.lobby.server.userDB.DbUserController' import. Should be before 'games.strategy.engine.lobby.server.userDB.HashedPassword'."